app/gui/view/debug_scene/component-group/src/lib.rs

```diff
@@ -162,6 +162,15 @@ fn create_component_group(
     component_group
 }
 
+fn create_wide_component_group(app: &Application) -> component_group::wide::View {
+    let component_group = app.new_view::<component_group::wide::View>();
+    component_group.set_width(450.0);
+    component_group.set_position_x(-200.0);
+    let background_color = color::Rgba(0.9, 0.91, 0.89, 1.0);
+    component_group.set_background_color(background_color);
+    component_group
+}
```

Comment:
A lot of hardcoded values. Refactor to constants

Reply:
I removed `background_color`. (it is no longer needed after merging with develop)
I don't think moving the other two literals (-200.0 and 450.0) into constants makes sense. They are purely example values and will probably last only until the next version of the demo scene.